src/devtools/client/webconsole/components/FilterBar/FilterBar.js

```diff
@@ -188,31 +176,31 @@ class FilterBar extends Component {
         active: filter[FILTERS.ERROR],
         label: getLabel(l10n.getStr("webconsole.errorsFilterButton.label"), FILTERS.ERROR),
         filterKey: FILTERS.ERROR,
-        dispatch,
```

Comment:
Above on line 160 we can get rid of dispatch and get the filterToggle from props, otherwise filter buttons don't work:
```
const { filter, filteredMessagesCount, toggleFilter } = this.props;
```